### PR TITLE
fix methods.py ->NameError: name 'sys' is not defined

### PR DESCRIPTION
**Pull request by crasy1**
fix 
```
scons: *** [core\version_hash.gen.windows.editor.x86_64.o] NameError : name 'sys' is not defined
Traceback (most recent call last):
  File "D:\Scoop\apps\scons\4.4.0\scons-local-4.4.0\SCons\Taskmaster.py", line 235, in execute
    self.targets[0].build()
  File "D:\Scoop\apps\scons\4.4.0\scons-local-4.4.0\SCons\Node\__init__.py", line 755, in build
    self.get_executor()(self, **kw)
  File "D:\Scoop\apps\scons\4.4.0\scons-local-4.4.0\SCons\Executor.py", line 384, in __call__
    return _do_execute_map[self._do_execute](self, target, kw)
  File "D:\Scoop\apps\scons\4.4.0\scons-local-4.4.0\SCons\Executor.py", line 117, in execute_action_list
    status = act(*args, **kw)
  File "D:\Scoop\apps\scons\4.4.0\scons-local-4.4.0\SCons\Action.py", line 1160, in __call__
    return act(
  File "D:\Scoop\apps\scons\4.4.0\scons-local-4.4.0\SCons\Action.py", line 1229, in __call__
    return c.__call__(self, target, source, env, *args, **kw)
  File "D:\Scoop\apps\scons\4.4.0\scons-local-4.4.0\SCons\Action.py", line 704, in __call__
    stat = self.execute(target, source, env, executor=executor)
  File "D:\Scoop\apps\scons\4.4.0\scons-local-4.4.0\SCons\Action.py", line 977, in execute
    result = spawn(shell, escape, cmd_line[0], cmd_line, ENV)
  File "methods.py", line 482, in mySpawn
    rv = mySubProcess(cmdline, env)
  File "methods.py", line 457, in mySubProcess
    if sys.version_info >= (3, 7, 0):
NameError: name 'sys' is not defined
```

### CODE REVIEW

**Comment by yedpodtrzitko** [comment]
Sorry about that. It was removed in #64784, which was created before another PR which needed this `import`.

File: methods.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import os
+import os,sys
```
Comment (aaronfranke) [inline methods.py:1]: Why not like this?

```suggestion
import os
import sys
```

**Comment by akien-mga** [comment]
Looks good! Could you squash the commits? See [PR workflow](https://docs.godotengine.org/en/latest/community/contributing/pr_workflow.html#the-interactive-rebase) for instructions.

**Comment by crasy1** [comment]
like this?sorry,this is my first PR

**Review by aaronfranke — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks! And congrats for your first merged Godot contribution :tada:

**Pull request merged**